### PR TITLE
Reorder RenderContext initializer to avoid -Wreorder

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: dbee4c48c14ac513e38b9993b43eafddb7de3690 available at: https://gist.github.com/83235a8f509ad0748ff7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6729.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6729.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: df78f895a67f810357766082ff103c912c39110a available at: https://gist.github.com/83235a8f509ad0748ff7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6729.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6729.dmg

File: libraries/render/src/render/Engine.h [hunk 1]
```diff
@@ -91,15 +91,15 @@ class RenderContext {
...
+    glm::vec4 _deferredDebugSize;
 
 protected:
     RenderArgs* _args;
 
     // Options
-    int _drawStatus = 0; // bitflag
-    bool _drawHitEffect = false;
+    int _drawStatus; // bitflag
+    bool _drawHitEffect;
```
Comment (ZappoMan) [inline libraries/render/src/render/Engine.h]: did you mean to remove these?

Comment (zzmp) [inline libraries/render/src/render/Engine.h]: Yes - this is only ever used through an initializer that sets these values, so I thought having "default" values was confusing.

**Pull request merged**